### DISCUSSION TITLE
oidc flow

### DISCUSSION THREAD

**Discussion by khyberkanwar** [Budibase/budibase#7408] [Help] [answered]
Hi, 
We are new to Budibase and have used OIDC support in Budibase to enable SSO with KeyCloak for our deployment.

Our solution requirement states that the user must be silently redirect to KeyCloak if they don't have active session against Budibase i.e. we DON'T want the user to see the Login page and click on the Login button.

Is there a way for us to enable the silent redirect behavior mentioned above via configuration? Please advise.

We need this silent redirect in conjunction with deep linking Budibase apps as we plan to iframe Budibase apps inside our web portal.

Your help will be much appreciated.

Thanks.
Khalid

**Comment by andz-bb** [comment, marked as answer]
Hey @khyberkanwar, at the minute this isn't possible in Budibase but we do have an open issue that I think covers your use case. You can indicate your interest and see the progress of the issue here: #4859 

Hopefully this issue is picked up soon for you!
